solc/CommandLineInterface.cpp

```diff
@@ -61,12 +61,15 @@
 #include <iostream>
 #include <fstream>
 
-bool g_HasOutputed = false;
-#define cout (g_HasOutputed = true, cout)
-
 using namespace std;
 namespace po = boost::program_options;
 
+bool g_hasOutputed = false;
+std::ostream& Out() {
```

Comment:
Should start with a lowercase letter.

Reply:
lowercase conflict with local variables. I will try some other name with lower case.